ontology/yaml/resources/units/units.yaml

```diff
@@ -101,6 +101,11 @@ distance:
     offset: 0
 electriccapacitance:
   farads: STANDARD
+electriccharge:
+  milliampere_hours: STANDARD
+  ampere_hours:
+    multiplier: 1000
```

Comment:
Please revise this multiplier to convert from the standard unit of coulomb to whatever target unit you need.